st_package_reviewer/check/file/check_os_system_calls.py

```diff
@@ -0,0 +1,31 @@
+from . import FileChecker
+import ast
+
+class CheckOsSytemCalls(FileChecker):
+    """Checks for any calls to os.system and suggests to use subprocess.check_call intead."""
+
+    def check(self):
+        self.counter = 0
+        self.visit_all_pyfiles()
+        if self.counter > 0:
+            self.warn("Detected os.system usage: Make sure you thought about the platform key in "
```

Comment:
This check warns about two things:

1. Suggest to use subprocess.check_output or Default.exec.ExecCommand
2. If such cases were found, it's likely the case that there is platform-dependent code. So I issue a warning at the end of the check telling the author that he or she needs to consider the platform key in the pull request.

With this in mind, do you really want the warning message about the platform key removed?